lib/fizzy/execute.cpp

```diff
@@ -476,6 +477,16 @@ inline uint64_t popcnt64(uint64_t value) noexcept
 {
     return static_cast<uint64_t>(__builtin_popcountll(value));
 }
+
+std::optional<uint32_t> find_export(
+    const Instance& instance, ExternalKind kind, std::string_view name)
+{
+    const auto it = std::find_if(instance.module.exportsec.begin(), instance.module.exportsec.end(),
+        [kind, name](const auto& export_) { return export_.kind == kind && export_.name == name; });
+
+    return (it == instance.module.exportsec.end() ? std::nullopt : std::make_optional(it->index));
```

Comment:
My preference is to have the positive case first.
```suggestion
    return (it != instance.module.exportsec.end()) ? std::make_optional(it->index) : std::nullopt;
```

Reply:
Depends on the definition of positive... But I don't care, changed.